### ISSUE TITLE
Question: Has anyone tried LoRA trigger words with SDXL words?

### ISSUE THREAD

**Issue by hylarucoder** [comfyanonymous/ComfyUI#1528]
The problem I encountered is as follows:

• ComfyUI + SD1.5 + LoRA + Trigger works fine.
• ComfyUI + SDXL + LoRA + Trigger doesn’t output images as expected. Just like A1111 LoRA is loaded but the trigger words are not written pic.

Has anyone ever generated LoRA-compliant images using SDXL and trigger words?

Here is my workflow

https://files.catbox.moe/3ksgk3.png

**Comment by thewayur** [comment]
u must use SDXL lora for SDXL Checkpoints

**Comment by hylarucoder** [comment]
@thewayur in fact, a use SDXL lora for SDXL Checkpoints. 

![微信截图_20230917172511](https://github.com/comfyanonymous/ComfyUI/assets/5625783/929c01dc-406c-46da-838e-59c6a06acd7b)

anything i do wrong?

**Comment by mcmonkey4eva** [comment]
You have to connect the "Model" output from "Load LoRA" to the model input on KSampler

**Comment by hylarucoder** [comment]
> You have to connect the "Model" output from "Load LoRA" to the model input on KSampler

@mcmonkey4eva thx for reply, still not working like a1111 does

<img width="1435" alt="Screenshot 2023-09-17 at 21 22 19" src="https://github.com/comfyanonymous/ComfyUI/assets/5625783/483cdbb6-5b06-4086-9712-742bae045870">



these images are generated from a1111

https://twitter.com/hylarucoder/status/1694656156978065635

**Comment by mcmonkey4eva** [comment]
can you try setting seed to 'fixed' (so it doesn't change) and do multiple generations with vs without the lora, but otherwise everything else (prompt/seed/etc) the same?
If results are identical: something's going wrong
If results are different: the lora is working, it's just not the results you expect, and other parameters might be relevant (is the prompt breaking because you wrote `womana` instead of I assume `woman`? Is that default negative prompt having unexpected effects? Is seed/sampler/... being different the cause?)

**Comment by hylarucoder** [comment]
I lock seed, prompts

then i click queue prompts. the result still not as a1111.

<img width="1327" alt="image" src="https://github.com/comfyanonymous/ComfyUI/assets/5625783/09e3ef93-897c-453e-b056-66bdcba16d44">

then i click queue prompts, it does not actually queue(queue size still 0), then refresh page, still can not queue.

**Comment by ltdrdata** [comment]
> I lock seed, prompts
> 
> then i click queue prompts. the result still not as a1111.
> 
> <img alt="image" width="1327" src="https://user-images.githubusercontent.com/5625783/268505897-09e3ef93-897c-453e-b056-66bdcba16d44.png">
> then i click queue prompts, it does not actually queue(queue size still 0), then refresh page, still can not queue.
> 
> I change seed control after gen to increment, it queued correctly

You need to define the problem accurately: whether you are aiming to achieve the same results as A1111, or if you want Lora to function. To obtain results identical to A1111, various settings need to be adjusted. Even if you make the seed identical, the default ComfyUI generates noise differently, so it won't yield the same results as A1111.

**Comment by hylarucoder** [comment]
@ltdrdata Thx for reply.

I trained a character LoRA with SDXL, the trigger words are "**lucasgirl**". When I use the loaded LoRA, it indeed has a certain impact on the image. But this impact is the same as when no trigger word is used in a1111.

**Expect face** (face in my training data, and generated from a111)

![image](https://github.com/comfyanonymous/ComfyUI/assets/5625783/28f4966c-caba-40b5-baf6-6b6e32688cc7)

However, when I use the comfy UI, **not once did it resemble the face I trained**. (looks very like one generated from a1111 without a trigger word)

<img width="891" alt="image" src="https://github.com/comfyanonymous/ComfyUI/assets/5625783/c86d3baf-1b70-4f27-88bd-fc718391a742">

**Comment by ltdrdata** [comment]
In the `Inspire pack`, there is a ksampler option that, same as A1111, provides the capability to generate seeds based on GPU. You can use this option to match the seed with A1111 and try generating it once.

![inspire](https://github.com/comfyanonymous/ComfyUI/assets/128333288/aa765c79-92a1-4435-aa20-606f684abf40)

https://github.com/ltdrdata/ComfyUI-Inspire-Pack

**Comment by hylarucoder** [comment]
Thank you for your patient answers, I really like your works

I continued my research for a while, and I think it may have something to do with the captions I used during training. When I only use "lucasgirl" and "woman", the face looks like this (whether on a1111 or comfyui).

![image](https://github.com/comfyanonymous/ComfyUI/assets/5625783/be1880fa-9646-405c-8a1f-7cb8acf79660)

But if I use more prompts, the face matches my training set.

```bash
body painting of woman,pale skin,sportswear,running,wide_shot,upper_body,looking at viewer,. human canvas,vibrant,detailed,transformative,expressive,
```
<img width="1196" alt="image" src="https://github.com/comfyanonymous/ComfyUI/assets/5625783/809441e9-7d72-43b1-9ff2-487bd33803b6">


I'm going to reassess my LoRA-training approach, thank you very much for your answers.
